spec/vcr/order_options/order_options_update_spec_vcr.yml

```
@@ -2,7 +2,7 @@
 http_interactions:
 - request:
     method: get
-    uri: https://cmr.sit.earthdata.nasa.gov/search/order-options.umm_json?id=848DA05B-51A2-1F3D-6783-6C27E5EA74B4&page_size=2000&provider_id=MMT_2
+    uri: https://cmr.sit.earthdata.nasa.gov/search/order-options.umm_json?concept_id=848DA05B-51A2-1F3D-6783-6C27E5EA74B4&page_size=2000&provider_id=MMT_2
```

```suggestion
    uri: https://cmr.sit.earthdata.nasa.gov/search/order-options.umm_json?concept_id=OO1200459850-CMR_ONLY&page_size=2000&provider_id=MMT_2
```
I don't have any issue with it but, maybe we should change the mock to look more like the real thing.

---

agree, fixed